snake/graphics.py

```diff
@@ -12,10 +12,15 @@ def drawTile(x, y, tile='', color=None):
 
     x = x * 2 + stage.padding[3] * 2 + stage.width / 2
     y += stage.padding[0] + stage.height / 2
-
-    screen.addstr(y, x, tile, color)
+    try:
+        screen.addstr(y, x, tile, color)
+    except curses.error:
+        pass
```

Comment:
> Errors should never pass silently.
> Unless explicitly silenced.

Source: `import this` ([PEP 20](https://www.python.org/dev/peps/pep-0020/))

_"There must be a better way!"_ ... though it could be that this is somewhat the best way. If you're Dutch. :smiley: